### PR TITLE
Ticket Model

### PR DESCRIPTION
**Pull request by kacperzolkiewski**

### CODE REVIEW

**Review by mateuszCabala95 — APPROVED** [review]

File: src/models/ticketModel/TicketModel.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    seatNumber: {
+        required: true,
+        type: Number,
+    },
+    cinemaHall: {
+        required: true,
+        type: Number,
+    },
+    QR_CODE: {
```
Comment (jkopyto) [inline src/models/ticketModel/TicketModel.ts:22]:
```suggestion
    qrCode: {
```

**Review by jkopyto — CHANGES_REQUESTED** [review]

File: src/models/ticketModel/TicketModel.ts (hunk 1 above)
Comment (kacperzolkiewski) [reply src/models/ticketModel/TicketModel.ts]: done

File: src/models/userModel/User.interface.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export default interface IUser {
```
Comment (jkopyto) [inline src/models/userModel/User.interface.ts:1]: `/src/models/userModel/User.interface.ts` -> `/src/models/user/User.model.ts`

ticket analogicznie

**Review by jkopyto — CHANGES_REQUESTED** [review]

File: src/models/userModel/User.interface.ts (hunk 2 above)
Comment (kacperzolkiewski) [reply src/models/userModel/User.interface.ts]: done

**Review by jkopyto — APPROVED** [review]

**Pull request merged**